### PR TITLE
chore: add tests for did with ewc chainname for siwe

### PR DESCRIPTION
**Pull request by nichonien**

### CODE REVIEW

**Review by artursudnik — APPROVED** [review]

File: test/LoginStrategy.siwetoken.test.ts [hunk 1]
```diff
@@ -131,6 +132,45 @@ it('Should authenticate issuer signature with Siwe', async () => {
...
+  expect(token).toBeTruthy();
+
+  await loginStrategy?.validate(
+    signature,
+    sampleSiwePayloadWithEwcChainID,
+    (_, user) => {
+      const decodedVerifiedUser = decode(user as string) as {
+        [key: string]: string | object;
+      };
+      console.log(decodedVerifiedUser);
```
Comment (jrhender) [inline test/LoginStrategy.siwetoken.test.ts:165]: IMO, I would remove the console log from the tests
```suggestion
```

File: test/LoginStrategy.siwetoken.test.ts [hunk 2]
```diff
@@ -131,6 +132,45 @@ it('Should authenticate issuer signature with Siwe', async () => {
...
+    version: '1',
+    nonce: 'bTyXgcQxn2htgkjJn',
+    issuedAt: '2022-01-27T17:09:38.578Z',
+    chainId: 246,
+    expirationTime: '2100-01-07T14:31:43.952Z',
+  });
+  const signature = await wallet.signMessage(
+    sampleSiwePayloadWithEwcChainID.prepareMessage()
+  );
+  expect(token).toBeTruthy();
```
Comment (jrhender) [inline test/LoginStrategy.siwetoken.test.ts:156]: Where is `token` from? Is it relevant here?

**Review by jrhender — COMMENTED** [review]
A couple comments

File: test/LoginStrategy.siwetoken.test.ts (hunk 2 above)
Comment (nichonien) [reply test/LoginStrategy.siwetoken.test.ts:156]: My bad, will update this.

**Review by jrhender — APPROVED** [review]

**Review by JGiter — APPROVED** [review]

**Pull request merged**